mozilla-central/setup

```diff
@@ -46,7 +49,102 @@ pushd $GIT_ROOT
 git pull
 popd
 
+date
+
+# Edit this to e.g. try.revision.ee64db93dcc149da9313460317257b8c42eec5b2 or whatever
+# to test other revisions. If you do this, also override INDEXED_GIT_REV below to some sane
+# git-equivalent of the revision because most likely it won't find anything in the mapfile.
+# This defaults to the most recent searchfox indexing run for mozilla-central.
+REVISION=mozilla-central.latest
+
+pushd $INDEX_ROOT
+wget -nv https://index.taskcluster.net/v1/task/gecko.v2.$REVISION.firefox.linux64-searchfox-debug/artifacts/public/build/target.json
+INDEXED_HG_REV=$(python $MOZSEARCH_PATH/scripts/read-json.py target.json moz_source_stamp)
+INDEXED_GIT_REV=$(awk -v REV=$INDEXED_HG_REV '$2 ~ REV { print $1 }' git_hg.map)
```

Comment:
Oh wow, I'm not looking forward to debug this, my awk-foo sucks :)

Reply:
basic awk is well worth learning, I find it super useful :)